lib/shared/messages.js

```diff
@@ -78,6 +78,13 @@ module.exports = {
         'The image does not appear to contain a partition table,',
         'and might not be recognized or bootable by your device.'
       ].join(' ')
+    },
+
+    largeDriveSize: (drive) => {
+      return [
+        'Drive <%= drive.description %> (<%= drive.device %>) is unusually large for an SD card or USB stick.',
```

Comment:
This needs to use string interpolation, as these are not lodash templates anymore